src/shared_modules/utils/tests/hashHelper_test.cpp

```diff
@@ -83,3 +89,25 @@ TEST_F(HashHelperTest, HashHelperHashIterativeSha256)
     EXPECT_EQ(sizeof(expected), result.size());
     EXPECT_TRUE(!memcmp(expected, result.data(), result.size()));
 }
+
+/**
+ * @brief Test the hashing of a file.
+ *
+ */
+TEST_F(HashHelperTest, HashFile)
+{
+    // SHA1SUM as string: 9c8ff5a046322fd04e244ce92e991118f26403f1
+    const std::vector<unsigned char> expectedHash {156, 143, 245, 160, 70, 50, 47,  208, 78, 36,
+                                                   76,  233, 46,  153, 17, 24, 242, 100, 3,  241};
```

Comment:
We could use hex literals here to make it easier to compare with the hash of the comment above:
```suggestion
const std::vector<unsigned char> expectedHash {0x9c, 0x8f,...
```

Reply:
Done!